osuapi/osu.py

```diff
@@ -131,7 +131,7 @@ def get_beatmaps(self, *, since=None, beatmapset_id=None, beatmap_id=None, usern
             A `str` representing the user's username, or an `int` representing the user's id.
             If specified, restrict results to a specific user.
         mode : :class:`osuapi.enums.OsuMode`
-            The osu! game mode for which to look up. Defaults to osu!standard.
```

Comment:
This should specify that passing None searches all beatmaps, and only if given restricts to a specific gamemode.

Reply:
How do you want it worded? I thought that "Defaults to any" was expressive enough.

Reply:
We can make it consistent at least, "If specified, restrict results to a specific osu! game mode".

A lot of the documentation here should be Optional[...] since None is valid, hmm